### ISSUE TITLE
Under preference-shortcuts-keybindings several keys yield -

### ISSUE THREAD

**Issue by Chris-N-K** [napari/napari#6111]
## 🐛 Bug

If I try to change the keybindings under preferences-shortcuts several keys lead to the same sign.
German layout: . - + lead to -
English layout: . - = lead to -

## To Reproduce

Steps to reproduce the behaviour:

1. open napari preferences shortcut menu
2. select a keybinding to change
3. type the mentioned keys

## Expected behavior

Get the key your pressed.

## Environment

 - Please copy and paste the information at napari info option in help menubar here:
```
napari: 0.4.18
Platform: Linux-5.14.0-1054-oem-x86_64-with-glibc2.35
System: Ubuntu 22.04.2 LTS
Python: 3.10.11 | packaged by conda-forge | (main, May 10 2023, 18:58:44) [GCC 11.3.0]
Qt: 5.15.6
PyQt5: 5.15.7
NumPy: 1.24.3
SciPy: 1.10.1
Dask: 2023.5.0
VisPy: 0.12.2
magicgui: 0.7.2
superqt: unknown
in-n-out: 0.1.7
app-model: 0.1.4
npe2: 0.7.0

OpenGL:
- GL version: 4.6 (Compatibility Profile) Mesa 22.2.5-0ubuntu0.1~22.04.3
- MAX_TEXTURE_SIZE: 16384

Screens:
- screen 1: resolution 1920x1080, scale 1.0

Settings path:
- /home/cnk/.config/napari/fim_d969ce57ebdec0a0facec3650983237d48847a92/settings.yaml
```
 - Any other relevant information:

**Comment by psobolewskiPhD** [comment]
I can reproduce on 0.4.18 but not on main (f8fa80a7d9c91081344dd3f9fda3c59283f4a661), so it appears to be fixed already.

**Comment by dalthviz** [comment]
Just checking issues but, seems like this issue should be closed? I think the automatic issue closing feature on GitHub only works if the base branch of the PR is the main branch so since the merged PR #6163 uses the `v0.4.19x` this was left open?

**Comment by psobolewskiPhD** [comment]
Good catch, thanks!
❤️
